Header header

float64 time
float64 ecef_x
float64 ecef_y
float64 ecef_z
float64 ecefV_x
float64 ecefV_y
float64 ecefV_z
float64 position_acc
float64 speed_acc
float64 p_DOP
float64 num_SV
float64 gps_fix_status



